schedule/yast/lvm/lvm_thin_provisioning_opensuse.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: lvm_thin_provisioning
+description: >
+  Complete OS deployment with unencrypted LVM drive management. Test creates
+  2 LVM and BIOS boot partitions. Thin pool and thin lv resides on the second
```

Comment:
s/resides on the second LVM partition/reside on the second and third PVs/
Make sense? we have a lot of mess with this description headers, if you could double check that matches your expectation would be great. Now we are going to start to reuse test module for multiple purposes driven by test data, the only source which will remain will be the yaml file.

Reply:
Done